pkg/schemas/streamlit.go

```diff
@@ -0,0 +1,84 @@
+package schemas
+
+import (
+	"fmt"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/helpers"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// DescribeStreamlitSchema represents output of DESCRIBE query for the single streamlit.
+var DescribeStreamlitSchema = map[string]*schema.Schema{
+	"name": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"title": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"root_location": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"main_file": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"query_warehouse": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"url_id": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"default_packages": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"user_packages": {
+		Type:     schema.TypeList,
+		Elem:     &schema.Schema{Type: schema.TypeString},
+		Computed: true,
+	},
+	"import_urls": {
+		Type:     schema.TypeList,
```

Comment:
Are those (this field and the one above) fields Lists or should be Sets?

Reply:
Changed to sets.